2024-10-22 22:34:56:INFO>  로그인에 성공했습니다
2024-10-22 22:35:04:INFO>  
2024-10-22 22:35:04:INFO>  start < ㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡ 계좌 정보 수집 ㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡ
2024-10-22 22:35:04:INFO>  계좌번호: 8087918911의 정보를 수집합니다
2024-10-22 22:35:04:INFO>  예수금 정보를 요청합니다
2024-10-22 22:35:04:INFO>  잔고 정보를 요청합니다
2024-10-22 22:35:05:INFO>  미체결 정보를 요청합니다
2024-10-22 22:35:06:INFO>  거래정보를 요청합니다
2024-10-22 22:35:06:INFO>  ㅡㅡㅡㅡㅡㅡㅡㅡㅡ 계좌 정보 수집완료 ㅡㅡㅡㅡㅡㅡㅡㅡㅡ > end
2024-10-22 22:35:06:INFO>  
2024-10-22 22:35:06:ERROR>  Traceback (most recent call last):
  File "C:\AutoTrading2\windows\main_win\main_win.py", line 232, in loginBtnClicked
    raise ValueError
ValueError

